### PR TITLE
Fix example latex

### PR DESCRIPTION
**Pull request by Kakulukian**
Fixing the misleading example for LaTex, used a workaround with html entities to avoid the parsing

### CODE REVIEW

**Review by julien-c — COMMENTED** [review]
ha yes ok i cannot read the doc it seems, i hadn't seen the parenthesis

**Comment by Kakulukian** [comment]
> ha yes ok i cannot read the doc it seems, i hadn't seen the parenthesis

https://moon-ci-docs.huggingface.co/docs/hub/pr_905/en/model-cards#can-i-write-latex-in-my-model-card it'll be less confusing

**Review by julien-c — APPROVED** [review]
smart

**Comment by julien-c** [comment]
i would also add it somewhere in https://huggingface.co/docs/hub/en/repositories-pull-requests-discussions @Kakulukian

**Review by julien-c — APPROVED** [review]

File: docs/hub/model-cards.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Yes! The Hub uses the [KaTeX](https://katex.org/) math typesetting library to re
 
 You have to use the following delimiters:
 - `$$ ... $$` for display mode
-- `\\` `(` `...` `\\` `)` for inline mode (no space between the slashes and the parenthesis).
+- `&#92;&#92;(...)\\` for inline mode (no space between the slashes and the parenthesis).
```
Comment (patrickvonplaten) [inline docs/hub/model-cards.md:136]: Gosh that took me ages to figure - out this is not correct I think. It should be `\\)`

**Pull request merged**